alembic/versions/26f00289fb9c_added_prev_upcoming_sprint_session_table.py

```diff
@@ -0,0 +1,74 @@
+"""Added prev,upcoming,sprint session table
+
+Revision ID: 26f00289fb9c
+Revises: 
+Create Date: 2024-03-05 00:04:56.427771
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '26f00289fb9c'
+down_revision: Union[str, None] = None
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('previous_sessions',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('topic', sa.String(), nullable=False),
+    sa.Column('owner', sa.String(), nullable=False),
+    sa.Column('duration', sa.String(), nullable=False),
+    sa.Column('date', sa.Date(), nullable=False),
+    sa.Column('slides', sa.String(), nullable=True),
+    sa.Column('recording', sa.String(), nullable=False),
+    sa.Column('description', sa.String(), nullable=True),
+    sa.Column('chat_log', sa.String(), nullable=True),
+    sa.Column('tags', sa.String(), nullable=True),
+    sa.Column('thumbnails', sa.String(), nullable=False),
+    sa.PrimaryKeyConstraint('id')
+    )
+
+    op.create_table('upcoming_sessions',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('topic', sa.String(), nullable=False),
+    sa.Column('owner', sa.String(), nullable=False),
+    sa.Column('duration', sa.String(), nullable=False),
+    sa.Column('date', sa.Date(), nullable=True),
+    sa.Column('notes', sa.String(), nullable=True),
+    sa.Column('event', sa.String(), nullable=True),
+    sa.PrimaryKeyConstraint('id')
+    )
+
+    op.create_table('sprint_session',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('topic', sa.String(), nullable=False),
+    sa.Column('owner', sa.String(), nullable=False),
+    sa.Column('duration', sa.String(), nullable=False),
+    sa.Column('date', sa.Date(), nullable=False),
+    sa.Column('slides', sa.String(), nullable=True),
+    sa.Column('recording', sa.String(), nullable=False),
+    sa.Column('description', sa.String(), nullable=True),
+    sa.Column('chat_log', sa.String(), nullable=True),
+    sa.Column('tags', sa.String(), nullable=True),
+    sa.Column('thumbnails', sa.String(), nullable=False),
+    sa.PrimaryKeyConstraint('id')
```

Comment:
We might benefit from an index on date, as we are querying the tables sorted by date. Though I am not 100% sure on this one.

Reply:
Would just adding an attribute of `index=True` on the column work? I don't have too much experience with this ^

Reply:
Yes, I think that will work.